libc/fuzzing/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=fuzzer")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=fuzzer,address")
```

Comment:
https://llvm.org/docs/LibFuzzer.html#fuzzer-usage :

_In most cases you may want to combine libFuzzer with [AddressSanitizer](https://clang.llvm.org/docs/AddressSanitizer.html) (ASAN), [UndefinedBehaviorSanitizer](https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html) (UBSAN), or both._

Reply:
Let's do that in a separate patch